code/game/gamemodes/clock_cult/clock_structures/ocular_warden.dm

```diff
@@ -134,3 +134,15 @@
 	target = null
 	visible_message("<span class='warning'>[src] settles and seems almost disappointed.</span>")
 	return 1
+
+/obj/structure/destructible/clockwork/ocular_warden/get_efficiency_mod(increasing)
+	if(GLOB.ratvar_awakens)
+		return increasing ? 0.5 : 2
+	. = 1 //Increasing isn't used here since it resets the value to 1 automatically when caluclating target damage
```

Comment:
specifically, if you set increasing to TRUE, `1/get_efficiency_mod(TRUE)` must have the same result as `1*get_efficiency_mod(FALSE)`